Jenkinsfile

```diff
@@ -79,6 +70,14 @@ elifePipeline {
             }
         }
 
+        stage 'Browser Tests', {
+            actions['test:browser'] = {
```

Comment:
`actions` is a dictionary of functions to be executed by `parallel`, so if you want to execute the browser tests here you have to remove this assignment and just use the contents of the `{    withCommitStatus(...)    }` 0-argument function